dpnp/dpnp_iface_arraycreation.py

```diff
@@ -1830,7 +2632,7 @@ def tril(x1, /, *, k=0):
     Examples
     --------
     >>> import dpnp as np
-    >>> np.tril([[1,2,3],[4,5,6],[7,8,9],[10,11,12]], -1)
+    >>> np.tril([[1, 2, 3], [4, 5, 6], [7, 8, 9], [10, 11, 12]], -1)
```

Comment:
We have to use `k=-1` or modify the function signature to make it similar to NumPy.
```suggestion
    >>> m = np.array([[1, 2, 3], [4, 5, 6], [7, 8, 9], [10, 11, 12]])
    >>> np.tril(m, k=-1)
```